feat(content): Patir 2 (#9959)
endless-sky/endless-sky: data/kahet/beyond patir assets.txt

Reviewer: Do we declare ships, outfits, fleets and effects all in one file?

Author: Currently I've done it like this given how it shows up in game and the limited file size, though splitting it up (or at least adding headers to each section for ease of view) would be trivial

		"Asteroid Young 4"

ship "Asteroid Large 1"
	"display name" "???"
	"noun" "asteroid"
	sprite "ship/asteroid adult"
	attributes
		category "Transport"
		"uncapturable" 1
		"inscrutable" 1
		"cost" 1350000000
		"hull" 136561
		"required crew" 1
		"bunks" 1
		"mass" 1174
		"drag" 70.4
		"heat dissipation" 10
		"fuel capacity" 1000
		"energy capacity" 1
		"fuel generation" 0.3
		"cargo space" 40
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
		"hull repair rate" 47
		"cargo scan power" 1
		"cargo scan efficiency" 0.001
		"thrust" 340.2
		"flare sprite" "effect/asteroid adult flare"
			"frame rate" 1
		"turn" 5931.4
		"asteroid mount jd" 1
		"asteroid mount am" 1
		"asteroid mount" 1
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile"
		"Asteroid Weapon Large" 2
		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	gun 0 0 "Asteroid Weapon Large"
	gun 0 0 "Asteroid Weapon Large"
	turret 0 0 "Asteroid Anti-Missile"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> ship "Asteroid Large 1" "Asteroid Large 2"

Reviewer 2: What's the reason for making every ship a variant of the first one?

Author: The original idea was to have them all have the same name ("???") but since display name exists it's now unnecessary. I will change this with the rest in a few days

	"final explode" "final explosion medium"

ship "Asteroid Large 1" "Asteroid Medium"
	"noun" "asteroid"
	sprite "ship/asteroid medium"
	attributes
		category "Transport"
		"uncapturable" 1
		"inscrutable" 1
		"cost" 950000000
		"hull" 91248
		"required crew" 1
		"bunks" 1
		"mass" 762
		"drag" 55.1
		"heat dissipation" 10
		"fuel capacity" 1000
		"energy capacity" 1
		"fuel generation" 0.3
		"cargo space" 40
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
		"hull repair rate" 47
		"cargo scan power" 1
		"cargo scan efficiency" 0.001
		"thrust" 327.2
		"flare sprite" "effect/asteroid medium flare"
			"frame rate" 1
		"turn" 4816.3
		"asteroid mount jd" 1
		"asteroid mount am 2" 1
		"asteroid mount 2" 1
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile 2"
		"Asteroid Weapon Large 2"
		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	gun 0 0 "Asteroid Weapon Large 2"
	turret 0 0 "Asteroid Anti-Missile 2"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> ship "Asteroid Large 1" "Asteroid Young"

Reviewer 2: `Asteroid Large 1` but only `Asteroid Young` instead of `Asteroid Young 1`?

		"drag" .1
		"acceleration" 1
		"lifetime" 30
		"submunition" "asteroid missile" 15
			"facing" 30
		"submunition" "asteroid missile" 15
			"facing" -30

outfit "Asteroid Jump Drive"
	category "Systems"
	cost 29631456
	thumbnail "outfit/asteroid weapon 7"
	"mass" 20
	"outfit space" -20
	"jump speed" 0.2
	"jump fuel" 200
	"jump drive" 1
	"asteroid mount jd" -1
	"jump effect" "jump drive circle"
	"jump effect" "jump drive spark 1"
	"jump effect" "jump drive spark 2"
	"jump effect" "jump drive spark 3"
	"jump effect" "jump drive spark 4"
	"jump effect" "jump drive spark 5"
	"jump effect" "jump drive spark 6"
	"jump effect" "jump drive spark 7"
	"jump effect" "jump drive spark 8"
	"jump effect" "jump drive spark 9"
	"jump effect" "jump drive 1"
	"jump effect" "jump drive 2"
	"jump effect" "jump drive 3"
	"jump effect" "jump drive 4"
	"jump effect" "jump drive 5"
	"jump effect" "jump drive 6"
	"jump effect" "jump drive 7"
	"jump effect" "jump drive 8"
	"jump effect" "jump drive 9"
	"jump effect" "jump drive 10"
	"jump effect" "jump drive 11"
	"jump effect" "jump drive 12"
	"jump effect" "jump drive 13"
	"jump effect" "jump drive 14"
	"jump effect" "jump drive 15"

> 	description "Whatever the mechanism that hides beneath the translucent orb may be, it functions similarly to a jump drive, bridging different parts of the universe together for the cost of merely 200 units of fuel."

Reviewer 2: Heh.

> # The "ajump drive" effect doesn't exist, but serve to reduce the number of effects spawned: as it always shows the same overall number, by showing empty ones it effectively reduces the visual bloat during the jump

Reviewer 2: I'm thinking we can change the jump effect code to allow this to be done in a cleaner manner.

Author: Mhm, this was just to adhere to the current limitations

		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 5"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 6"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 7"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 8"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 9"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

outfit "Asteroid Weapon Large 4"
	category "Guns"
	cost 57165239
	thumbnail "outfit/asteroid weapon 4"
	"mass" 33
	"outfit space" -33
	"weapon capacity" -33
	"gun ports" -1
	"asteroid mount 4" -1
	weapon
		sprite "effect/asteroid projectile print"
			"frame rate" 21
		"velocity" 100
		"drag" .1
		"acceleration" 1
		"lifetime" 20
		"turn" -6

> 		"submunition" "wep1 print" 2

Reviewer 2: Do you think these massive submunition chains could be improved if I finished #9832?

Author: I think so? Honestly, these weapon codes are so messy and made by so many parts (some of which taken from Fero and Todd) that I can't say for certain without testing

Reviewer 2: I'm fine with leaving these definitions where they're at if they're working already, but I'll revive that PR at some point and we can look at improving things here after it's merged.

# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

phrase "magic asteroid"
	word
		"???"

fleet "Magic Asteroid"
	government "Magic Asteroid"
	names "magic asteroid"
	cargo 1
	personality
		plunders harvests surveillance mute
	variant 11
		"Asteroid Large 1"
	variant 11
		"Asteroid Large 2"
	variant 11
		"Asteroid Large 3"
	variant 11
		"Asteroid Large 4"
	variant 11
		"Asteroid Large 5"
	variant 12
		"Asteroid Medium"
	variant 13
		"Asteroid Young 1"
	variant 13
		"Asteroid Young 2"
	variant 13
		"Asteroid Young 3"
	variant 13
		"Asteroid Young 4"

ship "Asteroid Large 1"
	"display name" "???"
	"noun" "asteroid"
	sprite "ship/asteroid adult"
	attributes
		category "Transport"
		"uncapturable" 1
		"inscrutable" 1

> 		"cost" 1350000000

Author: Changed to a varying price between 3 and 31 millions

		"hull repair rate" 47
		"cargo scan power" 1
		"cargo scan efficiency" 0.001
		"thrust" 340.2
		"flare sprite" "effect/asteroid adult 5 flare"
			"frame rate" 1
		"turn" 5931.4
		"asteroid mount jd" 1
		"asteroid mount am" 1
		"asteroid mount 2" 2
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile"
		"Asteroid Weapon Large 2" 2
		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	gun 0 0 "Asteroid Weapon Large 2"
	gun 0 0 "Asteroid Weapon Large 2"
	turret 0 0 "Asteroid Anti-Missile"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

ship "Asteroid Medium"
	"display name" "???"
	"noun" "asteroid"
	sprite "ship/asteroid medium"
	attributes
		category "Transport"
		"uncapturable" 1
		"inscrutable" 1
		"cost" 950000000
		"hull" 91248
		"required crew" 1
		"bunks" 1
		"mass" 762
		"drag" 55.1
		"heat dissipation" 10
		"fuel capacity" 1000
		"energy capacity" 1
		"fuel generation" 0.3
		"cargo space" 40
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95

> 		"hull repair rate" 47

Author: I think that, generally, it works better here for them to have the same hull repair across the board, with the capabilities of their weapons and the total hull varying more wildly (more on the latter later)

	"final explode" "final explosion medium"

ship "Asteroid Young 4"
	"display name" "???"
	"noun" "asteroid"
	sprite "ship/asteroid young 4"
	attributes
		category "Transport"
		"uncapturable" 1
		"inscrutable" 1
		"cost" 450000000
		"hull" 36462
		"required crew" 1
		"bunks" 1
		"mass" 591
		"drag" 39.7
		"heat dissipation" 10
		"fuel capacity" 1000
		"energy capacity" 1
		"fuel generation" 0.3
		"cargo space" 40
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
		"hull repair rate" 47
		"cargo scan power" 1
		"cargo scan efficiency" 0.001
		"thrust" 203.9
		"flare sprite" "effect/asteroid young 4 flare"
			"frame rate" 1
		"turn" 4019.1
		"asteroid mount jd" 1
		"asteroid mount small" 2
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile 2"
		"Asteroid Weapon Small"
		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	gun 0 0 "Asteroid Weapon Small"
	turret 0 0 "Asteroid Anti-Missile 2"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> outfit "Asteroid Anti-Missile"

Author: Given that this one will not be accessible to the player at any point of the campaign, I think that other than the outfit space changes I can leave this as-is

	category "Guns"
	cost 23460000
	thumbnail "outfit/asteroid weapon 3"
	"mass" 76
	"outfit space" -76
	"weapon capacity" -76
	"gun ports" -1
	"asteroid mount small" -1
	weapon
		sprite "projectile/asteroid weapon circle"
			"frame time" 20
			"start frame" 0
			"no repeat"
		sound "detainer"
		"hit effect" "large explosion"
		"inaccuracy" .5
		"submunition" "Asteroid Circle Submunition" 7
		"velocity" 5
		"lifetime" 100
		"reload" 60
		"firing force" 40
		"firing heat" 400
		"hit force" 3900
		"hull damage" 2400
		"slowing damage" -84
		"blast radius" 20
	description "Similar in appearance to the anti-missile used by some of these ships, this weapon was created with two functions in mind: slowing down an opponent until it gets close enough, and then eliminating it with heavy, direct damage to the hull."

outfit "Asteroid Circle Submunition"
	weapon
		sprite "projectile/asteroid weapon live"
		"inaccuracy" 360
		"hit effect" "detainer impact" 3
		"hit effect" "detainer smoke" 5
		"die effect" "tiny explosion"
		"turn" 6
		"homing" 3
		"tracking" 1
		"acceleration" 1.5
		"drag" 0.2
		"lifetime" 390
		"hull damage" 110
		"slowing damage" 12
		"hit force" -600
		"missile strength" 22

> outfit "Asteroid Weapon Large"

Author: The missile, renamed to "asteroid launch", now has 90 of missile strength while everything else has -90 where appropriate so that only the former will be treated as such by the anti missile. The missile itself now has 45 of hull damage, for a maximum of 400 damage per second if all 15 missiles hit. The mines (static) add 60 hull damage each, so that in case the player was so close to be hit by the mine printer as soon as it launched (something which requires being *very* close, range of just around 200 with no tracking), it would make a total of 940 hull damage per second

	"display name" "???"
	category "Guns"
	cost 8971535
	thumbnail "outfit/asteroid weapon 6"

> 	"mass" 24

Author: I had meant to fix them like two months ago, but here we are. Apologies, done now

		"turn" -8
		"lifetime" 5
		"submunition" "creeper13 static"
		"submunition" "creeper14 print"
		
outfit "creeper13 static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 15
		"submunition" "asteroid missile"

outfit "creeper14 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -8
		"lifetime" 5
		"submunition" "creeper14 static"
		"submunition" "creeper15 print"
		
outfit "creeper14 static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 10
		"submunition" "asteroid missile"

outfit "creeper15 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -8
		"lifetime" 5
		"submunition" "creeper15 static"
		
outfit "creeper15 static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 5
		"submunition" "asteroid missile"

> outfit "asteroid missile"

Reviewer 3: No damage?

	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"acceleration" 2
		"drag" .1
		"turn" 6
		"homing" 3
		"tracking" 1
		"lifetime" 120

> outfit "Asteroid Weapon Large 2"

Reviewer 3: If I haven't made a mistake in reading this (which is unlikely), this fires off a total of 31 `"asteroid laser 1"`s per shot. 31x1.7 = 52.7 discharge damage and 182.9 hull damage. The hull damage is very modest, the discharge damage less so (as it's effectively 5270 shield damage). At 0.4 shots/sec, this is a little over 1000 shield DPS - very strong, but certainly not unbalanceable.

	"display name" "???"
	category "Guns"
	cost 71670239
	thumbnail "outfit/asteroid weapon 1"
	"mass" 31
	"outfit space" -31
	"weapon capacity" -31
	"gun ports" -1
	"asteroid mount 2" -1
	weapon
		"velocity" 10
		"drag" .1
		"acceleration" 1

> 		"lifetime" 200000

Reviewer 3: ?!

		"lifetime" 550
		"inaccuracy" 5
		"split range" 180
		"submunition" "asteroid laser 1"
			"offset" 0 10

outfit "track4 print left"
	weapon
		sprite "projectile/asteroid weapon swarm"
			"frame rate" 5
			"random start frame"
		"acceleration" 1
		"drag" .1
		"turn" 20
		"live effect" "asteroid circle trail" 5
		"homing" 3
		"tracking" 1
		"lifetime" 550
		"inaccuracy" 5
		"split range" 180
		"submunition" "asteroid laser 1"
			"offset" 0 10

outfit "asteroid laser 1"
	weapon
		sprite "projectile/asteroid laser"
			"frame rate" 60
			"random start frame"
		sound "zapper"
		"hit effect" "skylance impact"
		"target effect" "asteroid target"
		"inaccuracy" .3
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"discharge damage" 1.7
		"hull damage" 5.9

effect "asteroid hit"
	sprite "effect/point defense hit"
		"frame rate" 13
		"no repeat"
	lifetime 1
	"velocity scale" 0

effect "asteroid target"
	sprite "effect/finisher sparks"
		"random start frame"
		"frame rate" 20
	"lifetime" 15
	"random angle" 180

effect "asteroid circle trail"
	sprite "effect/asteroid circle trail"
		"no repeat"
		"frame time" 4
	"lifetime" 20
	"random velocity" 5
	"random spin" 10
	"velocity scale" 0.3

> outfit "Asteroid Weapon Large 3"

Reviewer 3: This fires a total of 42 copies of the "asteroid missile". That's 1050 hull damage per shot, so a hull DPS of about 600? That's not unreasonable in itself (it's not reasonable for a weapon of only 16 tons, but the adapter should cover that).

The characteristics of the asteroid missile itself seem unreasonable, though - it's far too fast and too long-ranged.

Author: Reduced lifetime from 275 to 155 and acceleration from 2 to 1, while increasing turn from 20 to 60 to make it harder for them to miss

	"display name" "???"
	category "Guns"
	cost 98713449
	thumbnail "outfit/asteroid weapon 5"
	"mass" 16
	"outfit space" -16
	"weapon capacity" -16
	"gun ports" -1
	"asteroid mount 3" -1
	weapon
		sprite "projectile/asteroid twin projectile"
			"frame rate" 10
			"random start frame"
		"velocity" 20
		"drag" .1
		"acceleration" 1
		"lifetime" 30
		"submunition" "asteroid missile"
			"facing" 30
		"submunition" "asteroid missile"
			"facing" -30
		"submunition" "magic deployer1"
		"homing" 1
		"optical tracking" 1
		"turn" 0
		"split range" 180
		"reload" 100
		"firing heat" 5
	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy created by this weapon's fire continuously releases projectiles which home in onto enemies, until finally exploding and breaking up into an enormous barrage of similar projectiles. These attributes make it very good at overwhelming anti-missiles enough to render them useless, then impacting a fair share of damage while they are down."

> outfit "asteroid missile"

Reviewer 3: You have two submunitions defined with this name - this one, and the one at the end of the definition of `Asteroid Weapon Large`.

Author: I mentioned this earlier, but I changed the first one to "asteroid launcher". The two submunitions are similar, but it's easier to balance each separately

	weapon
		sprite "projectile/asteroid missile"
			"frame rate" 5
			"random start frame"

> 		"velocity" 50

Reviewer 3: This is probably too fast (as is the 66.6 speed from its acceleration and turning). 

Also, with 550 lifetime and a top speed of 66.6, this has a range of over 36 thousand, which is excessive.

		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 3"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 4"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 5"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 6"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 7"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 8"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 9"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

> outfit "Asteroid Weapon Large 4"

Reviewer 3: I like the idea behind this - a projectile that flies off and lays mines.

It probably ought to have some negative damage on the earlier projectiles so that just sticking it into the target at point-blank range doesn't immediately hit them with all 84 mines' worth of damage.

Author: Added negative corrosion, hull and slowing damage to each print. This makes it so only 22.9 hull damage, no corrosion and 1 slowing per mine are carried to the previous projectiles; this means hitting the very first projectile would impact nearly 2000 hull damage and 84 slowing damage, for a weapon that shoots every 130 frames

			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"

outfit "wep static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 575
		"turn" 360
		"homing" 2
		"tracking" 1
		"trigger radius" 240
		"blast radius" 400
		"discharge damage" 1.7
		"hull damage" 72.9
		"slowing damage" 10
		"hit force" 150
		"hit effect" "circle defense"

effect "circle defense"
	sprite "effect/explosion/nuke"
		"frame time" 60
		"random start frame"
	sound "point defense"
	lifetime 2

# Invisible ship in Lathia, represents the power of the asteroid base in the system

mission "Asteroid guarding Lathia"
	invisible
	landing
	to complete
		never
	npc save
		government "Magic Asteroid"
		personality forbearing frugal heroic staying vindictive
		system Lathia
		ship "Asteroid Planet" "Queen Asteroid"

ship "Asteroid Planet"
	attributes
		category "Heavy Warship"
		"uncapturable" 1
		"inscrutable" 1
		"cost" 1350000000
		"hull" 86135672193
		"required crew" 1
		"bunks" 1
		"mass" 674

> 		"drag" 30.4

Reviewer 3: Shouldn't these be much higher given the nominal size of this thing?

Or, if it is actually this size, it shouldn't have 86 times more hull than an Archon.

Author: I had chosen these values more to give the right look when moving (appearing somewhat inertialess, reaching the maximum speed almost instantly in any direction), but this can be easily kept by multiplying thrust and turn by the same amount; so, I've multiplied all these by a factor of 10

		"fuel capacity" 1000
		"energy capacity" 1
		"fuel generation" 0.3
		"cargo space" 40
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
		"hull repair rate" 47
		"cargo scan power" 1
		"cargo scan efficiency" 0.001
		"thrust" 203.9
		"flare sprite" "effect/asteroid young 4 flare"
			"frame rate" 1
		"turn" 4019.1
		"asteroid mount jd" 1
		"asteroid mount am 2" 1
		"asteroid mount small" 1
		weapon
			"blast radius" 80
			"shield damage" 800
			"hull damage" 400
			"hit force" 1200
	outfits
		"Asteroid Anti-Missile 2"
		"Asteroid Weapon Small"
		"Asteroid Jump Drive"
		
	engine 0 0
		zoom 1
		angle 0
		over
	gun 0 0 "Asteroid Weapon Small"
	turret 0 0 "Asteroid Anti-Missile 2"
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

outfit "Asteroid Anti-Missile"
	category "Turrets"
	cost 86346105
	thumbnail "outfit/asteroid weapon 6"
	"mass" 31
	"outfit space" -31
	"weapon capacity" -31
	"turret mounts" -1
	"asteroid mount am" -1
	weapon
		"fire effect" "asteroid am fire"
		"anti-missile" 420
		"velocity" 100
		"lifetime" 1
		"reload" 1
		"firing heat" 18

> 	description "Though its specifics are essentially the same as its smaller variant, its bubble shield is far bigger and just as indestructible. Despite becoming more instable in the process, this does not seem to have affected its capabilities in any way."

Reviewer 4: ```suggestion
	description "Though its specifics are essentially the same as its smaller variant, its bubble shield is far bigger and just as indestructible. Despite becoming more unstable in the process, this does not seem to have affected its capabilities in any way."
```

effect "asteroid am fire"
	sprite "effect/asteroid am fire"
		"frame time" 60
		"random start frame"
	sound "point defense"
	lifetime 2

outfit "Asteroid Anti-Missile 2"
	category "Turrets"
	cost 21280000
	thumbnail "outfit/asteroid weapon 8"
	"mass" 33
	"outfit space" -33
	"weapon capacity" -33
	"turret mounts" -1
	"asteroid mount am 2" -1
	weapon
		"hardpoint sprite" "hardpoint/asteroid am circle"
			"frame time" 30
		"fire effect" "asteroid am circle defense"
		"anti-missile" 420
		"velocity" 50
		"lifetime" 1
		"reload" 1
		"firing heat" 18

> 	description "A nearly impenetrable anti-missile field, instantly eliminating even the strongest missile that tries to pass through. However, the size of this protective bubble leaves greatly limits its capabilities."

Reviewer: ```suggestion
	description "A nearly impenetrable anti-missile field, instantly eliminating even the strongest missile that tries to pass through. However, the size of this protective bubble greatly limits its capabilities."
```

effect "asteroid am circle defense"
	sprite "effect/asteroid am circle defense"
		"frame time" 60
		"random start frame"
	sound "point defense"
	lifetime 2

outfit "Asteroid Weapon Small"
	category "Guns"
	cost 23460000
	thumbnail "outfit/asteroid weapon 3"
	"mass" 76
	"outfit space" -76
	"weapon capacity" -76
	"gun ports" -1
	"asteroid mount small" -1
	weapon
		sprite "projectile/asteroid weapon circle"
			"frame time" 20
			"start frame" 0
			"no repeat"
		sound "detainer"
		"hit effect" "large explosion"
		"inaccuracy" .5
		"submunition" "Asteroid Circle Submunition" 7
		"velocity" 5
		"lifetime" 100
		"reload" 60
		"firing force" 40
		"firing heat" 400
		"hit force" 3900
		"shield damage" 3200
		"hull damage" 2400
		"slowing damage" -84
		"blast radius" 20

> 	description "Similar in appearance to the anti-missile used by some of these ships, this weapon was created with two functions in mind: slowing down an opponent until he gets close enough, and then eliminate it with heavy, direct damage to the hull of its target."

Reviewer: ```suggestion
	description "Similar in appearance to the anti-missile used by some of these ships, this weapon was created with two functions in mind: slowing down an opponent until it gets close enough, and then eliminating it with heavy, direct damage to the hull."
```

		"target effect" "asteroid target"
		"inaccuracy" .3
		"velocity" 200
		"lifetime" 1
		"reload" 1
		"discharge damage" 1.7
		"hull damage" 5.9

effect "asteroid hit"
	sprite "effect/point defense hit"
		"frame rate" 13
		"no repeat"
	lifetime 1
	"velocity scale" 0

effect "asteroid target"
	sprite "effect/finisher sparks"
		"random start frame"
		"frame rate" 20
	"lifetime" 15
	"random angle" 180

effect "asteroid circle trail"
	sprite "effect/asteroid circle trail"
		"no repeat"
		"frame time" 4
	"lifetime" 20
	"random velocity" 5
	"random spin" 10
	"velocity scale" 0.3

outfit "Asteroid Weapon Large 3"
	category "Guns"
	cost 98713449
	thumbnail "outfit/asteroid weapon 5"
	"mass" 16
	"outfit space" -16
	"weapon capacity" -16
	"gun ports" -1
	"asteroid mount 3" -1
	weapon
		sprite "projectile/asteroid twin projectile"
			"frame rate" 10
			"random start frame"
		"velocity" 20
		"drag" .1
		"acceleration" 1
		"lifetime" 30
		"submunition" "asteroid missile"
			"facing" 30
		"submunition" "asteroid missile"
			"facing" -30
		"submunition" "magic deployer1"
		"homing" 1
		"optical tracking" 1
		"turn" 0
		"split range" 180
		"reload" 100
		"firing heat" 5

> 	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy created by this weapon's fire continuously releases projectiles which home in onto enemies, until finally exploding and breaking up into an enormous barrage of the same projectiles. These attributes make it very good at overwhelming anti-missiles enough to render them useless, then impacting a fair share of damage while they are down."

Reviewer: ```suggestion
	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy created by this weapon's fire continuously releases projectiles which home in onto enemies, until finally exploding and breaking up into an enormous barrage of similar projectiles. These attributes make it very good at overwhelming anti-missiles enough to render them useless, then impacting a fair share of damage while they are down."
```

	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"
		"submunition" "wep42 print"
	
outfit "wep42 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"

outfit "wep static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 575
		"turn" 360
		"homing" 2
		"tracking" 1
		"trigger radius" 240
		"blast radius" 400
		"discharge damage" 1.7
		"hull damage" 72.9
		"slowing damage" 10
		"hit force" 150
		"hit effect" "circle defense"

effect "circle defense"
	sprite "effect/explosion/nuke"
		"frame time" 60
		"random start frame"
	sound "point defense"
	lifetime 2

# Invisible ship in Lathia, represents the power of the asteroid base in the system

mission "Asteroid guarding Lathia"
	invisible
	landing
	to complete
		never
	npc save
		government "Magic Asteroid"

> 		personality heroic vindictive staying frugal forbearing

Reviewer 5: ```suggestion
		personality forbearing frugal heroic staying vindictive
```

		system Lathia
		ship "Asteroid Planet" "Queen Asteroid"

> ship "Asteroid Planet"

Author: No, it is meant to represent the power of the asteroid part of the Lathia system

	attributes

> 		category "Transport"

Reviewer 5: Sorry, but a transport with one bunk seems somewhat ridiculous to me.
```suggestion
		category "Medium Warship"
```
Or
```suggestion
		category "Heavy Warship"
```

		"drag" 0.2
		"lifetime" 390
		"hull damage" 50
		"slowing damage" 4
		"hit force" -300
		"missile strength" 22

outfit "Asteroid Weapon Large"
	"display name" "???"
	category "Guns"
	cost 18970000
	thumbnail "outfit/asteroid weapon 6"
	"mass" 43
	"outfit space" -43
	"weapon capacity" -43
	"gun ports" -1
	"asteroid mount" -1
	weapon
		sprite "effect/asteroid projectile print"
			"frame rate" 21
		"velocity" 10
		"drag" .1
		"lifetime" 20
		"turn" 16
		"submunition" "creeper1 print"
		"reload" 100
	description "By creating a circle of energy-based mines and then converting them into missiles at the same time, this weapon is capable of breaking through anti-missile defenses better than most."
	
outfit "creeper1 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 2
		"drag" .1
		"turn" -8
		"lifetime" 5
		"submunition" "creeper1 static"
		"submunition" "creeper2 print"
		
outfit "creeper1 static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 75
		"submunition" "asteroid launch"
		"hull damage" 60

> 		"missile strength" -90

Author: Removed the lines

		"hull damage" -50
		"slowing damage" -9
		"submunition" "wep static"
		"submunition" "wep42 print"
	
outfit "wep42 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"corrosion damage" -1.4
		"hull damage" -50
		"slowing damage" -9
		"submunition" "wep static"

outfit "wep static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 575
		"turn" 360
		"homing" 2
		"tracking" 1
		"trigger radius" 240
		"blast radius" 400
		"corrosion damage" 1.4
		"hull damage" 72.9
		"slowing damage" 10
		"hit force" 150
		"hit effect" "circle defense"

effect "circle defense"
	sprite "effect/explosion/nuke"
		"frame time" 60
		"random start frame"
	sound "point defense"
	lifetime 2

# Invisible ship in Lathia, represents the power of the asteroid base in the system

mission "Asteroid guarding Lathia"
	invisible
	landing
	to complete
		never
	npc save
		government "Magic Asteroid"
		personality forbearing frugal heroic staying vindictive
		system Lathia

> 		ship "Asteroid Planet" "Queen Asteroid"

Author: Yup, made it offer after Patir Mystery 8B
